r/vignettes/dataset.Rmd

```diff
@@ -322,9 +330,9 @@ by calling `write_dataset()` on it:
 write_dataset(ds, "nyc-taxi/feather", format = "feather")
 ```
 
-Next, let's imagine that the "payment_type" column is something we often filter on,
+Next, let's imagine that the `payment_type` column is something we often filter on,
 so we want to partition the data by that variable. By doing so we ensure that a filter like
-`payment_type == 3` will touch only a subset of files where payment_type is always 3.
+`payment_type == 3` will touch only a subset of files where `payment_type `is always 3.
```

Comment:
You didn't touch the example below that does 

```
ds %>%
  filter(payment_type == 3) %>%
  write_dataset("nyc-taxi/feather", format = "feather")
```

But that is no longer valid since we don't autocast numerics to strings anymore. The `group_by`/`write_dataset` will all work, but the query with a filter must have the quotes around it now

Reply:
I fixed this (sort of) in 88edc2d. It's an imperfect fix but I think it's about the best we can do without making major changes